modules/file-input/main/index.coffee

```diff
@@ -39,14 +49,15 @@ class FileInput extends hx.EventEmitter
       multiple: false
       dragEnabled: true
       buttonClass: 'hx-action'
-      noFilesText: 'No File Chosen'
-      filesSelectedText: 'Files Selected'
-      buttonText: 'Choose File'
+
+      buttonText: hx.userFacingText('fileInput', 'chooseFile')
```

Comment:
Generally I've gone with what the text is describing as the variable name, rather than where it will be used, I figured it would make more sense than having a localisation object that just says 'buttonText' - it's not clear what the button _does_ so it's not clear what text should be put in there